src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -347,6 +348,17 @@ public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.Reso
            Mandatory = false)]
         public bool? EnableSecureBoot { get; set; } = null;
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true)]
+        public SkuProfileVMSize[] VmSizes { get; set; }
```

Comment:
```suggestion
        public SkuProfileVMSize[] VmSize { get; set; }
```

Reply:
VmSize already exists, this is intended to differentiate from VmSize because it allows multiple VmSize**s**

Reply:
discussed with PM, updated to SkuProfileVmSize